packages/toolkit/src/query/core/buildInitiate.ts

```diff
@@ -258,7 +260,7 @@ Features like automatic cache collection, automatic refetching etc. will not be
     endpointDefinition: QueryDefinition<any, any, any, any>
   ) {
     const queryAction: StartQueryActionCreator<any> =
-      (arg, { subscribe = true, forceRefetch, subscriptionOptions } = {}) =>
+      (arg, { subscribe = true, forceRefetch, subscriptionOptions, forceQueryFn } = {}) =>
```

Comment:
This should be hidden from user space, so
```
// somewhere else
export const forceQueryFnSymbol = new Symbol
```
```suggestion
      (arg, { subscribe = true, forceRefetch, subscriptionOptions, [forceQueryFnSymbol]: forceQueryFn } = {}) =>
```